spec/lib/monitoring/middleware/prometheus_exporter_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'monitoring/middleware/prometheus_exporter'
```

Comment:
It's a nice-to-have, and not a requirement. I would be nice, if possible, to get rid of the `spec_helper` require here and instead have everything be loaded explicitly. 

I'm also wondering about `Rack::Test::Methods` vs whatever is used when testing rails controllers. They may be the same. If they're not I wonder what the difference is because we would want to test the `PrometheusExporter` in the context of rails and not in isolation.

Reply:
spec_helper can be removed if the following are explicitly loaded the exporter spec:
`require File.expand_path("../../../../../config/environment", __FILE__)`
`require 'rspec/rails'`
although the impact on loading times seems fairly small to me (13.1 -> 11.8 seconds on my machine), plus it ditches the SimpleCov functionality. Unclear to me if that's needed.

Rack::Test::Methods I guess is a more Rack-specific wrapper for testing API calls. It does not seem to be functionally much different than using the Rails request spec, i.e:
`describe Monitoring::Middleware::PrometheusExporter, type: :request do`
The latter works just fine so we can use that to be consistent with controllers.